src/main/java/org/broadinstitute/hellbender/tools/reference/ReferenceSequenceTable.java

```diff
@@ -0,0 +1,354 @@
+package org.broadinstitute.hellbender.tools.reference;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.broadinstitute.hellbender.engine.GATKPath;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceUtils;
+
+import java.sql.Ref;
+import java.util.*;
+
+public class ReferenceSequenceTable implements Iterable<ReferenceSequenceTable.TableRow> {
+
+    public static final String MISSING_ENTRY = "---";
+    public static final String MD5_COLUMN_NAME = "MD5";
+    public static final String LENGTH_COLUMN_NAME = "Length";
+    private static final int MD5_COLUMN_INDEX = 0;
+    private static final int LENGTH_COLUMN_INDEX = 1;
+    private Map<String, TableRow> tableByMD5;
```

Comment:
Add a blank line between the `static final` constants and your instance variables, for readability.